clappr/clappr-ios
# UI objects with a view property

### Goal
- UI objects shouldn't extend from `UIView`, they should contain a view property;
- `UIBaseObject` was renamed to `UIObject`;

### How to test
Run Clappr example and check if the video and media control are displayed properly.
